src/components/social-element/social-element.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers/utils";
+
+import * as styles from "./social-element.module.scss";
+
+const SocialElement = ({ className, links }) => {
+
+  return (
+    <div className={classnames(styles.container, className)}>
+      {links.map((item) => 
+        <a 
+          href={item.path} 
+          target="_blank" 
+          rel="noopener noreferrer" 
+          className={classnames(styles.socialIcon, {[styles.blogIcons]: item.id === 'blog'})}>
```

Comment:
why do you use id? maybe you can use `from`, what do you think?

Reply:
You can also avoid it all by passing the CSS class of the blog component

Reply:
What do you refer with passing the CSS class of the blog component? And, with from, do you refer an Array.from method or another thing?